client/frontend/client.go

```diff
@@ -643,3 +643,18 @@ func (c *clientImpl) getRandomClient() (workflowserviceclient.Interface, error)
 
 	return client.(workflowserviceclient.Interface), nil
 }
+
+func (c *clientImpl) GetReplicationTasks(
```

Comment:
suggestion: `GetReplicationTasks` -> `GetReplicationMessages`

Reply:
Thanks. I need lots of suggestion on names. Please keep them coming :)